### PR TITLE
Fix initialization order to avoid page-not-found error

### LINKED ISSUE #19456: App crashes and shows "Page not found" after loading (with slow internet connection and App extensions above a certain size)
### Describe the Bug

We experience a weird bug where the Admin App seems to crash after loading/hydration, when App extensions are imported that exceed a certain size and internet speed is slow. A bunch of users have already reported this issue to us.

After the crash, the App shows "Page not found" and no further navigation is possible. Even a Refresh can not recover the App in this state. To make the App work again, it is necessary to reroute the App to /admin to rerun the login procedure. If the network speed is sufficient then, the App will start to work again.

What we know so far:
- **Size of extensions matters**: It has something to do with the _total_ size of all extensions (not the number of extensions). It happens reproducibly for extensions with a total size of >300 KB. Whether modules, displays or interfaces are used, does not make any difference.
- **Network speed matters**: The issue is reproducibly between 5-15 Mbit. Interestingly, it seems to work on _very_ slow connections (like "Slow 3G" in Chrome). Above 30 Mbit it works flawlessly.

The console indicates that the App is loaded, e.g. "🕓 Application Loaded: 575.746826171875 ms". No further error logs or erroneous requests appear.

Any idea, what could cause this issue? Rollup maybe?

### To Reproduce

1. Extract attached archive
It is basically an empty Directus instance including a dummy interface with 164 KB size (that is imported twice to exceed 300KB)
2. Run `npm install` and then `npx directus start`
3. Open Chrome (115.0.5790.114) and navigate to http://localhost:8055
4. Login with admin@example.com and directus1234
5. Open network inspector and create a throttling preset with 5 Mbit
6. Refresh Admin App until it shows --> Page not found

[Directus-Package](https://1drv.ms/u/s!AjkAcRBi2bQbg5RDnxKQBOdSoJ0fHw?e=ABpqDy)

### Directus Version

v10.5.3

### Hosting Strategy

Self-Hosted (Custom)

### PR DESCRIPTION
**Pull request by christianrr**
Fixes #19456

### CODE REVIEW

File: contributors.yml [hunk 1]
```diff
@@ -70,3 +70,5 @@
 - AxeemHaider
 - alexvdvalk
 - thame
+- christianrr
+- cr-mutor
```
Comment (paescuj) [inline contributors.yml:74]: Let's only add the current username, we cannot 100% verify the other one
```suggestion
- christianrr
```

Comment (paescuj) [reply contributors.yml]: Thanks! 🚀

**Review by paescuj — APPROVED** [review]
Thanks @christianrr! Looks good to me 👍

**Review by DanielBiegler — APPROVED** [review]
Thank you for contributing! :heart:

**Pull request merged**